src/main/java/seedu/address/model/Inventory.java

```diff
@@ -66,8 +82,8 @@ public void removeIngredient(Ingredient key) {
 
     @Override
     public String toString() {
-        // TODO: After implementing Ingredients
-        return "";
+        return ingredients.asUnmodifiableObservableList().size() + " ingredients";
```

Comment:
is the toString() only used to print out the size?

Reply:
It's just copying the addressbook toString() style. Should we change it to display "Inventory: " + ingredients? (though on the user end this probably won't matter)